test/libsolidity/SemanticTest.cpp

```diff
@@ -197,6 +225,17 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 			bytes output;
 			if (test.call().kind == FunctionCall::Kind::LowLevel)
 				output = callLowLevel(test.call().arguments.rawBytes(), test.call().value.value);
+			else if (test.call().kind == FunctionCall::Kind::Builtin)
+			{
+				std::optional<bytes> builtinOutput{m_builtins[test.call().signature](test.call())};
```

Comment:
```suggestion
				std::optional<bytes> builtinOutput = m_builtins.at(test.call().signature)(test.call());
```
(I really cannot get used to brace-initialization for more complex things...)